src/components/MoviesList.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Suspense } from "react";
+import { useDispatch, useSelector } from 'react-redux'
+import { MoviesList, Movie } from '../assets/js/types'
+import { selectMovies } from "../store/reducers/movies"
+import {selectEditStatus, 
+        selectOpenForm, 
+        setOpenForm} from "../store/reducers/movies"
+import MovieCard from './blocks/MovieCard'
+const Modal = React.lazy(() => import('antd/lib/modal'))
+const FormBlock = React.lazy(() => import('./blocks/FormBlock'))
+
+const MoviesList: React.FC = () => {
+    const films = useSelector(selectMovies)
+    const isOpenForm = useSelector(selectOpenForm)
+    const isEdit = useSelector(selectEditStatus)
+    const dispatch = useDispatch()
+    
+    const closeFormModal = () => {
+        dispatch(setOpenForm(false))
+    }
+
+    const list = films.map((film: Movie) => {
+        return <MovieCard key={film.id} film={film} />
+    })
```

Comment:
You can move it to separate component `MovieCards`

Reply:
fixed it)